src/LearnMe.Core/Interfaces/Services/ICalendar.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using LearnMe.Core.DTO.Calendar;
+using LearnMe.Core.Services.Calendar.Utils;
+using LearnMe.Shared.Enum.Calendar;
+
+namespace LearnMe.Core.Interfaces.Services
+{
+    public interface ICalendar
+    {
+        Task<IEnumerable<CalendarEventDto>> GetAllEventsAsync(
+            int eventsPerPage,
+            int pageNumber,
+            string calendarId = Constants.CalendarId);
+
+        Task<CalendarEventDto> GetEventByIdAsync(int id);
+
+        Task<bool> CreateEventAsync(
+            CalendarEventDto eventData,
+            string calendarId = Constants.CalendarId,
+            string timezone = Constants.Timezone,
```

Comment:
Constanty raczej piszę się ze wszystkich duzych liter, np:

`Constants.CALENDAR_ID`
`Constatns.TIMEZONE`

Ale to nie jest duży błąd, kwestia gustu. Moim zapisem od razu widać, że masz do czynienia ze stałą :P